### ISSUE TITLE
Custom runMode?

### ISSUE THREAD

**Issue by KristijanZic** [serverpod/serverpod#2446]
I see we have `development.yaml`, `staging.yaml` and `production.yaml`. Would it be possible to add custom run modes like `alpha.yaml`, `beta.yaml`, `testing.yaml` etc?

**Comment by Isakdl** [comment]
You are not able todo that today, but we would like to add this functionality, this issue is tracked here: https://github.com/serverpod/serverpod/issues/2446

**Comment by KristijanZic** [comment]
@Isakdl I don't understand your answer. The issue is being tracked here (in the issue that I've opened) but you've just closed it?

**Comment by Isakdl** [comment]
@KristijanZic Sorry I pasted the wrong url, here is the correct one: https://github.com/serverpod/serverpod/issues/1212

**Comment by KristijanZic** [comment]
> @KristijanZic Sorry I pasted the wrong url, here is the correct one: https://github.com/serverpod/serverpod/issues/1212

Oh, makes sense, thank you :)
